samples/msal-angular-v2-samples/angular14-rxjs7-sample-app/src/app/app.module.ts

```diff
@@ -38,6 +38,7 @@ export function MSALInstanceFactory(): IPublicClientApplication {
       storeAuthStateInCookie: isIE, // set to true for IE 11. Remove this line to use Angular Universal
     },
     system: {
+      allowNativeBroker: false,
```

Comment:
nit: Comments missing, looks like all .ts and .js files had comments: 
 
`//Disables WAM  Broker`